# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/wr/WebRenderBridgeParent.h
# Commit: 741d451756e1
# Full Hash: 741d451756e187fccce7a18b20e7f93daeadedc3
# Author: Jamie Nicol <jnicol@mozilla.com>
# Date: 2020-10-09 15:35:54
# Regressor Bug: 1668878
# File Overlap Count: 1
# Description:
#   Bug 1668878 - Stop doing catch-up composites on webrender. r=mattwoodrow
#   
#   Currently, when webrender fails to render a frame on time, it will
#   attempt to catch up by rendering the next frame immediately (part way
#   through the vsync period) rather than delaying until the next
# ==============================================================================

diff -r 8b1680a2b965 -r 741d451756e1 gfx/layers/wr/WebRenderBridgeParent.h
--- a/gfx/layers/wr/WebRenderBridgeParent.h	Thu Oct 08 19:34:08 2020 +0000
+++ b/gfx/layers/wr/WebRenderBridgeParent.h	Thu Oct 08 19:16:00 2020 +0000
@@ -509,7 +509,6 @@
   CompositionOpportunityId mCompositionOpportunityId;
   nsCString mInitError;
 
-  VsyncId mSkippedCompositeId;
   TimeStamp mMostRecentComposite;
 
 #if defined(MOZ_WIDGET_ANDROID)
